---
layout: archive
title: "Education and Academic Background" #"CV and short bio"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}
<figure>
  <img src="/../../files/msu_baroda.jpg" alt="MSU Main Building" align="right" width="100" height="100">
  <figcaption> The Maharaja Sayajirao University of Baroda (2021) </figcaption>
</figure> <br>
<br> I have a Master’s degree in cell and molecular biology from Dr. Vikram Sarabhai Institute of Cell & Molecular Biology, The Maharaja Sayajirao University of Baroda, India.

**************************************************************************
<figure>
  <img src="/../../files/ncbs_mainbilding.jpg" alt="NCBS Main Building" align="left">
  <figcaption> NCBS Main Building (c) Eric Macwan (2018) </figcaption>
</figure>

## National Centre of Biological Sciences (NCBS), Bangalore, India  
Supervisor: <u> <a href="https://www.ncbs.res.in/faculty/sabari" target="_blank"> Dr. Sabarinathan Radhakrishnan </a> </u>  
Position: Graduate Trainee (August 2018 – July 2019)

In order to pursue my goals, I joined Dr. Sabarinathan Radhakrishnan’s lab at NCBS, Bangalore, India, after finishing my post-graduate studies. The Pan-cancer analysis of whole genomes confirmed that many driver mutations that affect tumour suppressor genes (TSGs) are two-hit inactivation events. For example, of the 954 tumours in the cohort with driver mutations in TP53, 77% had both alleles mutated. The study also shows that in a proportion of patients (TP53, 23%) driver mutations that affect TSGs are single-hit (monoallelic) inactivation events. These monoallelic driver events rejecting Knudson's two-hit theory of cancer causation can be explained by Dominant-negative mutations (DNMs), haploinsufficiency, or epistatic (interloci) interactions, where dominance arises from the existence of non-linear relationships between the genotypic and phenotypic values. DNMs in tumour suppressor genes have been reported in multiple cancers and they can drive tumour progression. To answer this, we focused on fishing and understanding the nature of DNMs in tumour suppressor genes (TSGs) across different tumour types using TCGA and PCAWG cancer datasets. Loss of function (LoF) mutations were considered, because similar to DNMs, they are heterozygous (monoallelic) in nature. A particular mutation can occur in the population as a monoallelic or biallelic driver event. Percentages were calculated for each mutation to identify mutations with a tendency to occur majorly (≥90%) as monoallelic events in the population. To consider these mutations as candidates for DNMs, I tried different approaches to evaluate them as potential pathogenic variants. I used Python and R/Bioconductor extensively in these analyses and also to visualise their localisation on different regions of genes. Using Bedtools, mutations falling in constrained coding regions (CCRs) were identified. CCRs are inferred to be under strong purifying selection and are enriched for known pathogenic variants (Havrilla et al., 2019). Even a single hit in CCRs will be sufficient to produce a pathogenic impact. Candidate mutations falling in CCRs become the ideal candidates as DNMs that can produce dominance as a monoallelic event. The study was carried further for these potential candidates to understand their position and function on their respective protein structures.

**************************************************************************
## Indian Institute of Science Education and Research (IISER), Pune, India  
Supervisor: Dr. Sudha Rajamani  
Position: Master’s Thesis Student (May 2017 – November 2017)

During my Master’s dissertation project in Dr. Sudha Rajamani’s lab at IISER, Pune, I studied “Binding of nucleobases to prebiotic Amphiphilic assemblies: Implications for the Origin of Life”. Similarly amphiphilic as phospholipids, fatty acids and related single-chain lipids are thought to have been available in the prebiotic environment. Only at a narrow pH range, fatty acids form stable bilayer membrane vesicles. These vesicle membranes might have played a crucial role as a selection pressure in the selection of present nucleobases. Nucleobases that were able to interact with the membrane were able to incorporate into the vesicles and accumulate at higher concentrations. This might have allowed for the formation of higher-order nucleobase structures leading to the formation of the first protocell. To test this hypothesis, I standardised a protocol for understanding interactions between fatty acid vesicles and nucleobases, and used Differential Interference Contrast (DIC) Microscopy to confirm vesicle formation. The protocol was then used to investigate the ability of different nucleobases (canonical and non-canonical) to interact and bind with fatty acid vesicles. Their interactions were calculated as the percentage of nucleobases that are interacting with vesicles and getting precipitated during ultracentrifugation along with vesicles. UV spectroscopy was used to check absorbance difference between control and test samples. In oleic acid vesicles, at pH 8.5, the prebiotically plausible nucleobases i.e. Barbituric acid, 2,4-Diamino-6-hydroxy-pyrimidine (DAHP), and 2,4,6-Triaminopyrimidine (TAP), show comparable bindings like their canonical bases, cytosine and uracil, respectively. The overall results indicate that with increasing concentration of the nucleobase, binding decreases for a specific concentration of fatty acid, with the exception of TAP in TDA assemblies. TAP, in comparison to cytosine, its close chemical relative, showed higher depletion and the possibility of forming higher-order structures in our reactions.

**************************************************************************
## Indian Institute of Science Education and Research (IISER), Pune, India  
Supervisor: Dr. Neelesh Dahanukar  
Position: Research Intern (May 2016 – July 2016)

During my summer internship in Dr. Neelesh Dahanukar’s lab at IISER, Pune, I made contributions to the project exploring the molecular phylogeny and phylogeography of freshwater fishes of sub-family Danioninae in India. The lab had a vast collection of preserved fish samples from all over the country spanning over 10 years. My project was divided into two parts. The first part focused on the genetic barcoding of the fishes. To achieve this feat, I dissected tissues from the fishes and extracted their DNA. I then amplified the barcoding gene (Cytochrome C Oxidase) using PCR and later purified it for sequencing. The second part of the project involved analysing the sequencing data. I performed sequence editing and alignment, phylogenetic hypothesis testing and phylogeography mapping. We were able to successfully chart out the distribution of 7 species from this family in the northeast and southwest regions of India. Molecular phylogenetic analysis of the barcoding gene helped us to identify new species, which are geographically and phylogenetically distant from known species. With regular interactions with my supervisor, my interest in evolutionary dynamics grew. I was able to learn about the fine balance of using wet lab and dry lab techniques to address questions studying evolutionary timescale.

**************************************************************************
## Institute of Mathematical Sciences (IMSC), Chennai, India  
Supervisor: Dr. Rahul Siddharthan  
Position: Research Intern (May 2015 – July 2015)

I studied computational prediction and characterisation of regulatory sites and cis-regulatory modules. The project involved prototype development (BLAST) and understanding the mathematical background of different alignment softwares. For prototype development, I used Python and probability-based models for predicting transcription factor (TF) binding sites.

**************************************************************************
My full CV is available to download [here](/../../files/CV-EricMacwan.pdf)
